Platforms/Ogre/OgrePlatform/src/MyGUI_OgreRenderManager.cpp

```diff
@@ -240,7 +321,14 @@ namespace MyGUI
 	{
 		if (mRenderSystem != nullptr)
 		{
-			mInfo.maximumDepth = mRenderSystem->getMaximumDepthInputValue();
+			if (mRenderSystem->getName() == "Direct3D11 Rendering Subsystem") // special case, it's not working with the value returned by the rendersystem
```

Comment:
I agree, but I'm not sure if this would break something else in ogre.
In Ogre 2.0, this is the only place this function is called.